ui/core/components/importers.tsx

```diff
@@ -285,10 +285,12 @@ export class Individual60UImporter<SpecType extends Spec> extends Importer {
 			if (itemJson.gems) {
 				itemSpec.gems = (itemJson.gems as Array<any>).filter(gemJson => gemJson?.id).map(gemJson => gemJson.id);
 			}
+			if (itemJson.suffixId) {
+				itemSpec.randomSuffix = itemJson.suffixId;
```

Comment:
Can we validate the suffix ID in browser?
Right now this will lead to a huge number of crash reports as I have yet to see a correct random suffix ID from 60 upgrades. Haven't testing manually setting it but no random suffix ID is correct when linking your character to your battle.net character.

I have already posted that in the cata bug channel on their discord.

Reply:
I just tested it with the provided import string using `-116` and that did work some of the other ones such as `-230` (feverflare) also work, and they match with the list we validate against in backend.

Sure we can add `suffixId` and `reforge` testing, but then we would also need to add a toast to inform the user to double check their gear.

Reply:
Intersting.

For example this belt https://www.wowhead.com/cata/item=63498/soul-breath-belt 60 upgrades will always give me -186 as the random suffix ID which is Feverflare and +154 Master and Haste. It should be 160 Mastery and Haste however.

So I assume it's just certain Items that are buged and not random suffixes in general. As even manually selecting the suffix will result in -186

Reply:
Our own UI is using ID -213 for feverflare for this belt. They can't both be right, can they? 🤔

Reply:
Just checked our DB:
```
{"id":-213,"name":"of the Feverflare","stats":[0,0,7889,4834,0,0,0,0,0,3320,0,0,0,0,3320,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,3320]},
{"id":-186,"name":"of the Feverflare","stats":[0,0,7889,4638,0,0,0,0,0,3205,0,0,0,0,3205,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,3205]},
```
They are both "valid" IDs, but the stats differ a bit. However it looks like the wowhead tooltip code actually has a way of "migrating" to the correct IDs.
Problem being that our code will not do that and indeed select the wrong stats. I think I should for now actually remove any  `reforge` if `randomSuffix` is set, and show a Toast after importing that we did so.

Reply:
Isnt this just “normal” and “heroic” versions of the random suffix? They should both be valid.

Reply:
Well an easy migration strategy would be to map them based on the name. There will never be two ID's with the same name valid for the same Item. So the UI seems to know which IDs are valid for an item as the selection only shows 4 valid ones.

Thus we could do `ID -> Lookup Name -> Lookup ID for Item based on Name`

Reply:
@InDebt Added the following Toast + removed the random suffix + reforge from 60U: 
![image](https://github.com/wowsims/cata/assets/1216787/a25d7ff3-3f2e-4310-b1d0-2e6b1da5b074)

Reply:
> Well an easy migration strategy would be to map them based on the name. There will never be two ID's with the same name valid for the same Item. So the UI seems to know which IDs are valid for an item as the selection only shows 4 valid ones.
> 
> Thus we could do `ID -> Lookup Name -> Lookup ID for Item based on Name`

We probably also need to migrate the reforges then right? I think for now showing this message and then in time create a proper migration would be the best way forward. @rosenrusinov @InDebt

Reply:
We should also update our `itemSpec` logic in `database.ts` because that just gets the suffix by ID no questions asked:
```
let randomSuffix: ItemRandomSuffix | null = null;
if (itemSpec.randomSuffix && !!this.getRandomSuffixById(itemSpec.randomSuffix)) {
	randomSuffix = this.getRandomSuffixById(itemSpec.randomSuffix)!;
}
```